### DISCUSSION TITLE
Campi non compilabili

### DISCUSSION THREAD

**Discussion by zux1848** [globaleaks/globaleaks-whistleblowing-software#3893] [Q&A]
Buonasera a tutti.
Ho creato un questionario, ma rilevo delle anomalie di comportamento che non riesco a comprendere:
In alcuni campi di testo (che siano singola linea o multilinea risultano non compilabili; Sia che siano condizionati dalla risposta precedenti che no.
Talvolta cancellando e ricreando il campo il problema si risolve, talvolta no. E non trovo una logica per la correzione del problema.
Qualcuno ha già avuto questa anomalia? come l'avete affrontata?

Grazie,

**Comment by evilaliv3** [comment]
Potete fornire un link alla piattaforma in oggetto o caricare un export del questionario così da supportare la riproducibilità della problematica?
